docs/proposals/koordlet/20241210-pod-resources-proxy .md

```diff
@@ -0,0 +1,114 @@
+---
+title: Pod Resources Proxy
+authors:
+- "@ZiMengSheng,@ferris-cx"
+reviewers:
+- "@hormes"
+- "@saintube"
+creation-date: 2024-12-10
+last-updated: 2025-02-17
+---
+# Pod Resources Proxy 
+
+## Motivation
+
+The Kubernetes community already provides the kubelet pod-resources endpoint, which allows third-party consumers to inspect the mapping between devices and pods. 
```

Comment:
Please add a more basic user scenario to clarify the original motivation of the proposal.